src/components/ServiceList.jsx

```diff
@@ -70,7 +70,7 @@ const ServiceList = () => {
               checked={isMostPopular}
             />
           </div>
-          <div className="col-md-9 col-xs-12">
+          <div className="col-md-9 col-xs-12 dg_search_header">
```

Comment:
@tmccoy529 I sent you a message regarding this part of the code, I don't think its needed in here, and should be addressed in the components project. Everything else looks good.

Reply:
Yeah i just saw this change. I can rip this all out and put int he components project. The 80px legend margin still needs to be removed though so we can just make this PR for that removal.